IGC/WrapperLLVM/include/llvmWrapper/IR/ConstantFolder.h

```diff
@@ -11,9 +11,11 @@ SPDX-License-Identifier: MIT
 
 #include "llvm/Config/llvm-config.h"
 #include "llvm/IR/ConstantFolder.h"
+#include <llvm/Support/Casting.h>
 
 namespace IGCLLVM
 {
+    using namespace llvm;
```

Comment:
Please remove the `using` directive from here and use explicit scoping throughout WrapperLLVM module.

Reply:
Done. And rebased.